src/registrar/config/settings.py

```diff
@@ -599,6 +599,21 @@
 SECRET_REGISTRY_HOSTNAME = secret_registry_hostname
 
 # endregion
+
+# region: Connection Pool----------------------------------------------------------###
+# Use this variable to set the size of our connection pool in client.py
+# WARNING: Setting this value too high could cause frequent app crashes!
+# Having too many connections open could cause the sandbox to timeout,
```

Comment:
This shouldn't happen now that we added in async communication. Are you still seeing this?